src/CliMenu.php

```diff
@@ -277,7 +297,7 @@ protected function drawMenuItem(MenuItemInterface $item, bool $selected = false)
 
         return array_map(function ($row) use ($setColour, $unsetColour) {
             return sprintf(
-                "%s%s%s%s%s%s%s\n\r",
```

Comment:
dropped this `\r` because we don't support windows anymore and phpstorm keep changing the line endings when I was editing the test files which was mega annoying.